### PR TITLE
Integration tests with real db

### LINKED ISSUE #97: Consumer unable to write a report to the db
When I upload an archive in the log I can see:
```
2020/02/18 15:53:47 Making connection to data storage, driver=postgres datasource=postgres://****:****@insights-results-aggregator-db.ccx-data-pipeline-ci.svc/aggregator?sslmode=disable
2020/02/18 15:53:47 Making connection to data storage, driver=postgres datasource=postgres://****:****@insights-results-aggregator-db.ccx-data-pipeline-ci.svc/aggregator?sslmode=disable
2020/02/18 15:53:47 Initializing HTTP server at :8080
2020/02/18 15:53:47 Starting HTTP server at :8080
2020/02/18 15:53:47 Consumer has been started, waiting for messages send to topic ccx.ocp.results
2020/02/18 15:54:04 Consumed message offset 640
2020/02/18 15:54:04 Results for organization 12383032 and cluster e8823fc9-337f-40b7-934c-2c90980fcd57
2020/02/18 15:54:04 Error writing report to database: pq: syntax error at or near "OR"
2020/02/18 15:54:04 Error processing message consumed from Kafka: pq: syntax error at or near "OR"
```

### PR DESCRIPTION
**Pull request by Sergey1011010**
# Description

Integration tests with both sqlite & postgres and some fixes so that tests don't faila

Fixes #97

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- This change requires a documentation update
- Refactor (refactoring code, removing useless files)
- Unit tests (no changes in the code)

## Testing steps

`make test` & `make integration_tests`

### CODE REVIEW

File: local_storage/dockerize_kafka.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# TODO: rewrite with docker-compose?
```
Comment (bond95) [inline local_storage/dockerize_kafka.sh]: Yes, please, we already has docker-compose with kafka https://github.com/RedHatInsights/ccx-data-pipeline/blob/master/docker-compose.yml

**Comment by quarckster** [comment]
Nice! I would like to propose to not include integration tests here. Let's use a dedicated suite for that and run tests in Jenkins.

**Comment by bond95** [comment]
@quarckster @tisnik if we will use another suit for integration tests, I think we can close this PR, WDYT?

**Comment by quarckster** [comment]
@bond95 this PR fixes #95 so just integration tests part should be removed.

**Comment by Sergey1011010** [comment]
@quarckster you mean #97?

**Comment by quarckster** [comment]
@Sergey1011010 yes, sorry

**Comment by bond95** [comment]
@quarckster oops, yes, sorry.

**Comment by Sergey1011010** [comment]
I think we can close this because fixes for the bug were moved here https://github.com/RedHatInsights/insights-results-aggregator/pull/113 and we decided to do integration tests in another repo

**Pull request closed without merging**